### PR TITLE
[Bug] [connector] Use Hudi source Serializable Exception

### PR DESCRIPTION
**Pull request by liujinhui1994**

### CODE REVIEW

**Comment by liujinhui1994** [comment]
@CalvinKirs  hello，seems to be timed out, can you help me?

**Comment by simon824** [comment]
hi @liujinhui1994  , you can trigger rebuild by yourself with these commands.
```
git commit --allow-empty -m 'trigger rebuild' 
git push
```

**Comment by xleoken** [comment]
hi @liujinhui1994, can you put the exception stackstrace?

File: seatunnel-connectors/seatunnel-connector-spark-hudi/src/main/scala/org/apache/seatunnel/spark/sink/Hudi.scala [hunk 1]
```diff
@@ -41,7 +41,7 @@ class Hudi extends SparkBatchSink {
   override def output(df: Dataset[Row], environment: SparkEnvironment): Unit = {
     val writer = df.write.format("org.apache.hudi")
     for (e <- config.entrySet()) {
-      writer.option(e.getKey, e.getValue.toString)
+      writer.option(e.getKey, config.getString(e.getKey))
```
Comment (simon824) [inline seatunnel-connectors/seatunnel-connector-spark-hudi/src/main/scala/org/apache/seatunnel/spark/sink/Hudi.scala]: Change to `String.valueOf(e.getValue.unwrapped())` is better.

File: seatunnel-connectors/seatunnel-connector-spark-hudi/src/main/scala/org/apache/seatunnel/spark/source/Hudi.scala [hunk 2]
```diff
@@ -36,7 +36,7 @@ class Hudi extends SparkBatchSource {
 
     val reader = env.getSparkSession.read.format("org.apache.hudi")
     for (e <- config.entrySet()) {
-      reader.option(e.getKey, e.getValue.toString)
+      reader.option(e.getKey, config.getString(e.getKey))
```
Comment (simon824) [inline seatunnel-connectors/seatunnel-connector-spark-hudi/src/main/scala/org/apache/seatunnel/spark/source/Hudi.scala]: ditto

File: seatunnel-connectors/seatunnel-connector-spark-hudi/src/main/scala/org/apache/seatunnel/spark/sink/Hudi.scala (hunk 1 above)
Comment (liujinhui1994) [reply seatunnel-connectors/seatunnel-connector-spark-hudi/src/main/scala/org/apache/seatunnel/spark/sink/Hudi.scala]: ok

File: seatunnel-connectors/seatunnel-connector-spark-hudi/src/main/scala/org/apache/seatunnel/spark/source/Hudi.scala (hunk 2 above)
Comment (liujinhui1994) [reply seatunnel-connectors/seatunnel-connector-spark-hudi/src/main/scala/org/apache/seatunnel/spark/source/Hudi.scala]: ok

**Comment by wuchunfu** [comment]
@liujinhui1994 Can you post your exception information?

**Comment by simon824** [comment]
> @liujinhui1994 Can you post your exception information?

#1273 cc @wuchunfu

**Review by wuchunfu — APPROVED** [review]
LGTM

**Comment by wuchunfu** [comment]
Well done, looking forward to your next contribution

**Pull request merged**